libs/design-system/search-autocomplete/src/lib/search-autocomplete.component.html

```diff
@@ -0,0 +1,45 @@
+<div class="search-autocomplete-container">
+  <mat-form-field class="search-field" subscriptSizing="dynamic">
+    <mat-label>{{ label() }}</mat-label>
+
+    <mat-icon matPrefix>search</mat-icon>
+
+    <input
+      matInput
+      type="text"
+      aria-label="Search with autocomplete"
+      [formControl]="searchControl"
+      [matAutocomplete]="auto"
+    />
+
+    @if (searchControl.value) {
+      <button matSuffix mat-icon-button aria-label="Clear search" type="button" (click)="clearSearch()">
+        <mat-icon>close</mat-icon>
+      </button>
+    }
+
+    <mat-autocomplete
```

Comment:
I've talked to Libby and this component won't show a list of options so this should be removed